package/Editor/Models/ModelsUI.cs

```diff
@@ -71,7 +95,8 @@ private void UpdatePage()
             
                 pageModels.Add(models[i]);
             }
-
+            numberPages = models.Count / maxModelsPerPage;
+            Debug.LogWarning("firstIndexImage: " + firstImageIndex + " " + models.Count + " " + maxModelsPerPage + " Count / maxModels = " + models.Count/maxModelsPerPage + " pageModels " + pageModels.Count);
```

Comment:
maybe you should remove this warning if it was only for easing debugging

Reply:
And same for this one

Reply:
are you sure you pushed the removing of the Debug.LogWarning ?

Reply:
Include in the commit from 2 days ago:
[fix: navigation in pages](https://github.com/scenario-labs/Scenario-Unity/pull/149/commits/62f61408115b73fab6ef9d7c7a40999f881fe183)